### PR TITLE
edgetest bump versions

### PR DESCRIPTION
**Pull request by fdosani**
Updating `pyproject.toml` and `requirements.txt`

### CODE REVIEW

**Comment by jdawang** [comment]
I don't want to approve my own changes so @gladysteh99 ?

File: pyproject.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ authors = [
   { name="Faisal Dosani", email="faisal.dosani@capitalone.com" },
 ]
 license = {text = "Apache Software License"}
-dependencies = ["boto3<=1.35.53,>=1.9.92", "PyYAML<=6.0.1,>=5.1", "pandas<=2.2.3,>=1.5.0", "numpy<=2.0.2,>=1.22.0", "polars>=0.20.0", "pyarrow>=10.0.1"]
+dependencies = ["boto3<=1.35.63,>=1.9.92", "PyYAML<=6.0.1,>=5.1", "pandas<=2.2.3,>=0.25.2", "numpy<=2.1.3,>=1.22.0", "polars>=0.20.0", "pyarrow>=10.0.1"]
```
Comment (gladysteh99) [inline pyproject.toml:9]: pyarrow backed datatype is only supported from pandas 1.5.0 onwards, are we comfortable with bumping the lower pin to 1.5.0?

Comment (jdawang) [reply pyproject.toml:9]: Yeah I'm comfortable with that. Even 1.5 is very old.

**Review by gladysteh99 — APPROVED** [review]
LGTM!

**Pull request merged**